modules/RTC/ortc/RTCPeerConnection.js

```diff
@@ -0,0 +1,532 @@
+        this._remoteStreams.clear();
+
+        // TODO: Close and emit more stuff.
+    }
+
+    /**
+     */
+    createOffer(options, callback, errback) {
+        logger.debug('createOffer() options:', options);
+
+        if (USE_PROMISES) {
+            return this._createOffer(options);
+        }
+
+        this._createOffer(options)
+            .then(desc => callback && callback(desc))
+            .catch(error => errback && errback(error));
+    }
+
+    /**
+     */
+    createAnswer(options, callback, errback) {
+        logger.debug('createAnswer() options:', options);
+
+        if (USE_PROMISES) {
+            return this._createAnswer(options);
+        }
+
+        this._createAnswer(options)
+            .then(desc => callback && callback(desc))
+            .catch(error => errback && errback(error));
+    }
+
+    /**
+     */
+    setLocalDescription(desc, callback, errback) {
+        logger.debug('setLocalDescription() desc:', desc);
+
+        if (USE_PROMISES) {
+            return this._setLocalDescription(desc);
+        }
+
+        this._setLocalDescription(desc)
+            .then(() => callback && callback())
+            .catch(error => errback && errback(error));
+    }
+
+    /**
+     */
+    setRemoteDescription(desc, callback, errback) {
+        logger.debug('setRemoteDescription() desc:', desc);
+
+        if (USE_PROMISES) {
+            return this._setRemoteDescription(desc);
+        }
+
+        this._setRemoteDescription(desc)
+            .then(() => callback && callback())
+            .catch(error => errback && errback(error));
+    }
+
+    /**
+     */
+    addIceCandidate(candidate, callback, errback) {
+        logger.debug('addIceCandidate() candidate:', candidate);
+
+        if (USE_PROMISES) {
+            return this._addIceCandidate(candidate);
+        }
+
+        this._addIceCandidate(candidate)
+            .then(() => callback && callback())
+            .catch(error => errback && errback(error));
+    }
+
+    /**
+     */
+    addStream(stream) {
+        logger.debug('addStream()');
+
+        this._addStream(stream);
+    }
+
+    /**
+     */
+    removeStream(stream) {
+        logger.debug('removeStream()');
+
+        this._removeStream(stream);
+    }
+
+    /**
+     */
+    createDataChannel() {
+        logger.debug('createDataChannel()');
+    }
+
+    /**
+     */
+    getLocalStreams() {
+        return Array.from(this._localStreams);
+    }
+
+    /**
+     */
+    getRemoteStreams() {
+        return Array.from(this._remoteStreams);
+    }
+
+    /**
+     */
+    getStats() {
+        // TODO
+    }
+
+    /**
+     *
+     */
+    _createIceGatherer(pcConfig) {
+        const iceGatherOptions = {
+            gatherPolicy: pcConfig.iceTransportPolicy || 'all',
+            iceServers: pcConfig.iceServers || []
+        };
+        const iceGatherer = new RTCIceGatherer(iceGatherOptions);
+
+        // NOTE: Not implemented by Edge.
```

Comment:
They should, and I expect we all prefer to be ready once they do that rather than just relying on currently inaccurate API implementation.